ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -60,6 +60,11 @@ export class Mint extends IronfishCommand {
         'Minimum number of block confirmations needed to include a note. Set to 0 to include all blocks.',
       required: false,
     }),
+    rawTransaction: Flags.boolean({
```

Comment:
Would be nice to have this output to a file. Passing a raw transaction as a CLI flag to the `wallet:post` is vary hard to get working locally for me. And if the transaction is too big it might not even be possible to paste it into the CLI. So taking a file path for the raw transaction would help a lot I think. I don't want to hold up this PR for this though but I think it should be followed up on